### DISCUSSION TITLE
Getting error when execute "  provisioner "local-exec" {     command = <<EOF cd /home/centos/roboshop24-ansible git pull sleep 30 ansible-playbook -i ${var.name}.msahu.online, main.yml -e ansible_user=centos -e ansible_password=DevOps321 -e component_name=${var.name} EOF   } }"

### DISCUSSION THREAD

**Discussion by Msahu201989** [learndevopsonline/learndevopsonline#1677] [Q&A]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/113416711/4e6d009f-fd83-4c60-b1ee-176a6b957b18)

│
│ TASK [frontend : Restart Nginx] ************************************************
│ fatal: [frontend.msahu.online]: FAILED! => {"changed": false, "msg": "Unable to start service nginx: Job for
│ nginx.service failed because the control process exited with error code.\nSee \"systemctl status nginx.service\" and
│ \"journalctl -xe\" for details.\n"}
│
│ PLAY RECAP *********************************************************************
│ frontend.msahu.online      : ok=6    changed=5    unreachable=0    failed=1
│ skipped=0    rescued=0    ignored=0
│
│
╵
╷
│ Error: local-exec provisioner error
│
│   with module.components["cart"].null_resource.ansible,
│   on module/main.tf line 27, in resource "null_resource" "ansible":
│   27:   provisioner "local-exec" {
│
│ Error running command 'cd /home/centos/roboshop24-ansible
│ git pull
│ sleep 30
│ ansible-playbook -i cart.msahu.online, main.yml -e ansible_user=centos -e ansible_password=DevOps321 -e
│ component_name=cart
│ ': exit status 2. Output: Already up to date.

**Comment by sreedharm07** [comment]
i dont find any ip from the info you given to check
